app/test_main.py

```diff
@@ -1,3 +1,36 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_cat_dog_age_zero_year():
+    ages = get_human_age(14, 14)
+    assert ages == [0, 0]
+
+
+def test_cat_dog_age_exactly_one_year():
+    ages = get_human_age(15, 15)
+    assert ages == [1, 1]
+
+
+def test_cat_dog_age_still_one_year():
+    ages = get_human_age(23, 23)
+    assert ages == [1, 1]
+
+
+def test_cat_dog_age_exactly_two_years():
+    ages = get_human_age(24, 24)
+    assert ages == [2, 2]
+
+
+def test_cat_dog_age_still_two_years():
+    ages = get_human_age(26, 26)
+    assert ages == [2, 2]
```

Comment:
```suggestion
    assert get_human_age(26, 26) == [2, 2]
```
You can simplify the test code